src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -14799,34 +14775,31 @@ class FindRelatedBooksAsyncFlattenedAsyncPagedAllOdyssey
    /// <summary>
    /// Testing calling forms
    /// </summary>
-   public static void SampleFindRelatedBooks()
+   public static async Task SampleFindRelatedBooksAsync()
    {
-       public static async Task SampleFindRelatedBooksAsync()
+       LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
+       IEnumerable<BookName> names = new[]
        {
-           LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-           IEnumerable<BookName> names = new[]
-           {
-               new BookName("[SHELF_ID]", "[BOOK_ID]"),
-           };
-           IEnumerable<ShelfName> shelves = new[]
-           {
-               new ShelfName("[SHELF_ID]"),
-           };
-           gax::PagedAsyncEnumerable<FindRelatedBooksResponse, BookName> response = libraryServiceClient.FindRelatedBooksAsync(names, shelves);
-           // Iterate over pages (of server-defined size), performing one RPC per page
-           foreach (BookName item in response)
-           {
-               // FIXME: inspect the results
-           }
+           new BookName("[SHELF_ID]", "[BOOK_ID]"),
+       };
+       IEnumerable<ShelfName> shelves = new[]
+       {
+           new ShelfName("[SHELF_ID]"),
+       };
+       await gax::PagedAsyncEnumerable<FindRelatedBooksResponse, BookName> response = libraryServiceClient.FindRelatedBooksAsync(names, shelves);
+       // Iterate over pages (of server-defined size), performing one RPC per page
+       foreach (BookName item in response)
```

Comment:
It is not possible to use a `foreach` loop over an asynchronous enumerable (unfortunately).

```
await response.ForEachAsync((BookNam item) =>
{
    // FIXME: inspect the results
});
```